content/en/docs/instrumentation/net/libraries.md

```diff
@@ -26,46 +26,51 @@ dotnet add package OpenTelemetry.Instrumentation.{library-name-or-type}
 It is typically then registered at application startup time, such as when
 creating a [TracerProvider](/docs/concepts/signals/traces/#tracer-provider).
 
+## Note on Versioning
+
+The Semantic Conventions (Standards) for attribute names are not currently
+stable therefore the instrumentation package is currently not in a released
+state. That doesn't mean that the functionality itself is not stable, only that
+the names of some of the attributes may change in the future, some may be added,
+some may be removed. This means that you need to use the `--prerelease` flag, or
+install a specific version of the package
+
 ## Example with ASP.NET Core and HttpClient
 
 As an example, here's how you can instrument inbound and output requests from an
 ASP.NET Core app.
 
-First, get the appropriate packages:
+First, get the appropriate packages of OpenTelemetry Core:
+
+```
+dotnet add package OpenTelemetry
```

Comment:
this line is not needed,

Reply:
The reason I left is that it pins the OpenTelemetry version for when people add instrumentation libraries in the next step. I feel this is a little safer given the state we're in with instrumentation libraries.

Reply:
I dont think it pins anything. IF an instrumentaion library brings a newer one, it'll be taken.

Reply:
it will have, say, 1.4.0 in for otel... then when you add instrumentation, if that requires 1.5.0-alpha, one of two things will happen... either you'll end up with an error/mismatch.... or you'll see explicitly that the core otel library has been updated to match... both of those are positive outcomes in my book.